tests/RobotFramework/tests/cumulocity/service_monitoring/service_monitoring.robot

```diff
@@ -137,41 +139,41 @@ Check if a service is up
     Custom Test Setup
     ThinEdgeIO.Start Service    ${service_name}
     ThinEdgeIO.Service Should Be Running    ${service_name}
-       
-    Device Should Exist                      ${DEVICE_SN}_${service_name}    show_info=False 
+
+    Device Should Exist                      ${DEVICE_SN}_${service_name}    show_info=False
     ${SERVICE}=    Cumulocity.Device Should Have Fragment Values    status\=up        timeout=${TIMEOUT}
-    
+
     Should Be Equal    ${SERVICE["name"]}    ${service_name}
     Should Be Equal    ${SERVICE["serviceType"]}    service
     Should Be Equal    ${SERVICE["status"]}    up
     Should Be Equal    ${SERVICE["type"]}    c8y_Service
     ThinEdgeIO.Stop Service    ${service_name}
     Custom Test Teardown
-    
+
 
 Check if a service is down
     [Arguments]    ${service_name}
     Custom Test Setup
     ThinEdgeIO.Start Service    ${service_name}
-    Device Should Exist                      ${DEVICE_SN}_${service_name}    show_info=False 
-    ThinEdgeIO.Stop Service    ${service_name}   
+    Device Should Exist                      ${DEVICE_SN}_${service_name}    show_info=False
+    ThinEdgeIO.Stop Service    ${service_name}
     ThinEdgeIO.Service Should Be Stopped  ${service_name}
-       
-    Device Should Exist                      ${DEVICE_SN}_${service_name}    show_info=False 
+
+    Device Should Exist                      ${DEVICE_SN}_${service_name}    show_info=False
     ${SERVICE}=    Cumulocity.Device Should Have Fragment Values    status\=down
-    
+
     Should Be Equal    ${SERVICE["name"]}    ${service_name}
     Should Be Equal    ${SERVICE["serviceType"]}    service
     Should Be Equal    ${SERVICE["status"]}    down
     Should Be Equal    ${SERVICE["type"]}    c8y_Service
-   
+
     Custom Test Teardown
 
 Check if a service using configured service type
     [Arguments]    ${service_name}
     Execute Command    tedge config set service.type thinedge
     Custom Test Setup
-    ThinEdgeIO.Start Service    ${service_name}
+    ThinEdgeIO.Restart Service    ${service_name}
```

Comment:
Why restart is needed here?, is it already started by the `setup`?

Reply:
The service starts when the connection to Cumulocity is not yet established, and fails with `c8y_api::http_proxy: An error occurred while retrieving internal Id, operation will retry in 60 seconds and mapper will reinitialize`. This error is mentioned earlier in the file, but bumping the timeout didn't really fix the problem. This did. If not having to restart the service is a hard requirement in this test, then I'll note it and be sure to find a solution that does not involve restarting, later.

Reply:
>  If not having to restart the service is a hard requirement in this test, then I'll note it and be sure to find a solution that does not involve restarting, later.

Restarting the service is okay.